sql/core/src/test/scala/org/apache/spark/sql/connector/DataSourceV2SQLSuite.scala

```diff
@@ -120,13 +120,13 @@ class DataSourceV2SQLSuite
   }
 
   test("DescribeTable extended using v2 catalog") {
-    spark.sql("CREATE TABLE testcat.table_name (id bigint, data string)" +
+    spark.sql("CREATE TABLE testcat.default.table_name (id bigint, data string)" +
       " USING foo" +
       " PARTITIONED BY (id)" +
       " TBLPROPERTIES ('bar'='baz')" +
       " COMMENT 'this is a test table'" +
       " LOCATION 'file:/tmp/testcat/table_name'")
-    val descriptionDf = spark.sql("DESCRIBE TABLE EXTENDED testcat.table_name")
+    val descriptionDf = spark.sql("DESCRIBE TABLE EXTENDED testcat.default.table_name")
```

Comment:
Does this also fix SHOW TABLE EXTENDED?

Reply:
There is no V2 version of `SHOW TABLE EXTENDED` yet.

Reply:
makes sense